pkg/authtoken/providers/azure/azure_msi.go

```diff
@@ -8,13 +8,13 @@ import (
 	"context"
 	"fmt"
 
+	"go.goms.io/fleet/pkg/authtoken"
```

Comment:
Move this to the bottom of the imports on all files. We like to group the imports as:
1. common standard library packages in Go
2. third-party (or external) packages
3. fleet packages
[Caravel Coding Style Guide](https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/231243/Caravel-Coding-Style-Guide?anchor=common-pr-feedbacks%3A)